README.rst

```diff
@@ -69,6 +69,12 @@ Enable auto capture event logs in your nameko configuration file:
     EVENTLOG_DISPATCHER:
       auto_capture: true
       entrypoints_to_exclude: []
+      event_types:
+        generic: log_event
+        entrypoint_fired: entrypoint_fired
```

Comment:
Thanks for the suggestion. I see how these different event types could be confusing.
It seems to me that the best solution for now will be to add the ability to provide an alternative Nameko event type in the config, which will be the event type to handle by the entrypoints.

```
EVENTLOG_DISPATCHER:
  auto_capture: true
  entrypoints_to_exclude: []
  event_type: log_event
```
 
The `entrypoint_fired` event type is still an event type, but not a Nameko event type and just an application level event type (for the services using the dependency) that we want to log and goes into the event metadata. This is not something that will affect how services interact with each other and will not be configurable in the config. For now, if we ever have the need to change it, we can always subclass:

```python
from nameko_eventlog_dispatcher import EventLogDispatcher

class MyEventLogDispatcher(EventLogDispatcher):
    ENTRYPOINT_FIRED = 'my_entrypoint_fired'
```